src/plugins/facebook/facebook-en.hbs

```diff
@@ -7,10 +7,17 @@
 	"tag": "facebook",
 	"parentdir": "facebook",
 	"altLangPrefix": "facebook",
-	"dateModified": "2015-08-03"
+	"dateModified": "2022-10-21"
 }
 ---
 <span class="wb-prettify all-pre hide"></span>
+<section class="alert alert-success">
```

Comment:
Please use `alert-info` instead of `alert-success`
```suggestion
<section class="alert alert-info">
```
In other words, we want to see a blue box with an info sign instead of a green box with a checkmark

Reply:
I changed the alert-success to an alert-info and it now displays correctly on both the English and French pages. It also closed the pull request for some reason? I'm not exactly sure why that happened